### PR TITLE
Solved the Task 

### PR DESCRIPTION
**Pull request by UbaidAfzal98**
### All Puzzle Solved

I have Solved all the Puzzles that was my task to do if i made mistakes 
plz let me know so that i can practice more on it to learn more.

### CODE REVIEW

**Comment by s-m-quadri** [comment]
@UbaidAfzal98 Bhai! :clap: Bhoot **Zabardast!**

File: puzzle-five.md [hunk 1]
```diff
@@ -19,7 +19,16 @@ Spotting patterns is important in writing good programs.
 
 Answers are as follows:
 
-__
+- The first light House Contains the value **36** becouse the series contains the difference of 5.
+<br>
```
Comment (s-m-quadri) [inline puzzle-five.md:23]: Ye `<br>` add karne ki zaraorat nahi thi.

File: puzzle-five.md [hunk 2]
```diff
@@ -19,7 +19,16 @@ Spotting patterns is important in writing good programs.
...
+- The Second Light House contains the value **28** becouse the series is like
+<br> 
+0, 0+1 = 1, 1+2 = 3, 3+3 = 6, 6+4 = 10, 10+5 = 15, 15+6 = 21, 21+7 = 28
+<br>
+- The Third light House Contains the value of **64** becouse the series is square root of Numbers.
+<br>
+- The last light house contians the value of **21** becouse it is fibonachi series in which every new number N is equal to **N = N + N-1**.
+
```
Comment (s-m-quadri) [inline puzzle-five.md:31]: Simply Aisa bhi likh sate hai.

```md
- The Second Light House contains the value **28** becouse the series is like
0, 0+1 = 1, 1+2 = 3, 3+3 = 6, 6+4 = 10, 10+5 = 15, 15+6 = 21, 21+7 = 28
- The Third light House Contains the value of **64** becouse the series is square root of Numbers.
- The last light house contians the value of **21** becouse it is fibonachi series in which every new number N is equal to **N = N + N-1**.
```

File: puzzle-two.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ Spotting differences/bugs helps programmers write better programs and solve prob
 <!-- Note: These are comments, will not show in Preview section. -->
 <!--   ... Thus, you can remove, it's just for your understanding -->
 
-I found at least one difference, i.e. ___.
+I found at least one difference, i.e. the second image contains the extra red dot inside the yellow portion of the Image.
```
Comment (s-m-quadri) [inline puzzle-two.md:24]: Good.

**Comment by UbaidAfzal98** [comment]
**@s-m-quadri  Jazakallah Khair**

**Review by s-m-quadri — COMMENTED** [review]
Just some suggestions.

**Comment by s-m-quadri** [comment]
_Bhoot acha kiya. Aap first hai, jinhone exercise karke send kiya hai. :laughing: Good!
_

**Comment by UbaidAfzal98** [comment]
@s-m-quadri still have many confusions like removing errors and looks so complicated
i hope with so much practicing it will be easy 
if you suggest some road map it will be your kindness

File: puzzle-two.md (hunk 3 above)
Comment (UbaidAfzal98) [reply puzzle-two.md:24]: App Profile b check karai ga mai nai README.md file jo aap nai banai thi edit kar di thodi bhut abi uss mai dalnai laik khuch kiya nhi tah too issi liyai kahli hai profile

**Comment by s-m-quadri** [comment]
# General Roadmap for learning

:muscle:  It just requires [practice, practice and a lot of practice](https://youtu.be/oGWX4xpUwr8)

:writing_hand: You can also refer to instructions I provided at #1 in [How? Section](https://github.com/s-m-quadri/learn-github/issues/1#issue-1987414415).

:1234: Count the number of hours you spend on learning and practicing. In first `50 hours` it will be very frustrating, in `next 50 hours` you got some motivation, and finally you'll get interest after completing `100 hours`.

> Finally,
>  :smiling_face_with_tear: Feel free to reach me if have any difficulties.

File: puzzle-two.md (hunk 3 above)
Comment (s-m-quadri) [reply puzzle-two.md:24]: _Ok._

Comment (s-m-quadri) [reply puzzle-two.md:24]: @UbaidAfzal98, Aap ki profile dekha huin. Achi banaye, **simple aur clear hai**. Abhi ke liye satisfactory hai.

**... Jaise jaise zarurat padhi, `important links`, `images` wagaira add karte rehna usme.**

Comment (UbaidAfzal98) [reply puzzle-two.md:24]: @s-m-quadri Ok bahtr Bhut Shukriya aap ka 👍

**Pull request merged**